src/js/control-bar/progress-control/seek-bar.js

```diff
@@ -116,8 +117,12 @@ class SeekBar extends Slider {
     }
 
     if (liveTracker && liveTracker.seekableEnd() === Infinity) {
```

Comment:
Perhaps we should run this code on a `seekableendchange` triggered by the `liveTracker`? That should be the only time this code needs to run.